README.md

```diff
@@ -1,22 +1,21 @@
 # KMK: Clackety Keyboards Powered by Python
 ![GitHub](https://img.shields.io/github/license/KMKfw/kmk_firmware)
 ![GitHub contributors](https://img.shields.io/github/contributors/KMKfw/kmk_firmware)
-[![Discord](https://img.shields.io/discord/493256121075761173?logo=Discord)](https://discord.gg/QBHUUpeGUd)
 ![Lines of code](https://img.shields.io/tokei/lines/github/KMKfw/kmk_firmware)
 ![GitHub issues](https://img.shields.io/github/issues-raw/KMKfw/kmk_firmware)
 ![GitHub closed issues](https://img.shields.io/github/issues-closed/KMKfw/kmk_firmware)
 
-Documentation available at [📖 kmkfw.io](http://kmkfw.io/).
```

Comment:
Bit of an unrelated change, but these are out of date and AFAIK unmaintained as BoardSource has largely moved on from KMK (to my knowledge). To avoid confusion, we'll just stop linking them for now.